src/test/CMakeLists.txt

```diff
@@ -33,7 +33,9 @@ add_subdirectory(journal)
 add_subdirectory(libcephfs)
 add_subdirectory(librados)
 add_subdirectory(librados_test_stub)
-add_subdirectory(libradosstriper)
+if(WITH_LIBRADOSSTRIPER)
+  add_subdirectory(libradosstriper)
+endif(WITH_LIBRADOSSTRIPER)
```

Comment:
@tchaikov Just to make sure I'm following: you're saying that newer cmake styles use: 
if(FOO)
 ...
endif()

...which is fine with me. I do want to avoid the case discussed by @BlaineEXE and @smithfarm above where someone comes along and goes "why's that one different". I'm fine with having it all normalized in a separate PR that just handles formatting and leaving it as a piecemeal step for now; I'm /also/ equally fine with leaving it and letting it match what's already there. It's really up to you.

Reply:
i insist.